bin/p2-inspect/main.go

```diff
@@ -112,7 +98,7 @@ func main() {
 	enc.Encode(statusMap)
 }
 
-func addKVPToMap(result kp.ManifestResult, source int, filterNode, filterPod string, statuses map[string]map[string]NodePodStatus) error {
+func addKVPToMap(result kp.ManifestResult, source int, filterNode, filterPod string, statuses map[string]map[string]inspect.NodePodStatus) error {
```

Comment:
Since you've moved the main data structure to its own module, would it make sense to move the functions that operate on the data structure to the module too?